Snowflake.Data/Core/SFSession.cs

```diff
@@ -222,6 +222,7 @@ internal void UpdateSessionParameterMap(List<NameValueParameter> parameterList)
             logger.Debug("Update parameter map");
             foreach (NameValueParameter parameter in parameterList)
             {
+                logger.Debug(parameter.name + " : " + parameter.value);

```

Comment:
I am not sure how expensive is it in .net or c#, but in java, string concat in logging is probably not a good idea? I should probably use DebugFmt or String format?

Reply:
Oups, this log line was not supposed to make it in the PR. It was there for me only. I'll just remove it completely.